EIPS/eip-5218.md

```diff
@@ -0,0 +1,252 @@
+---
+eip: 5218
+title: NFT Rights Management Standard
+description: An interface for creating copyright licenses that transfer with an NFT.
+status: Draft
+type: Standards Track
+category: ERC
+author: James Grimmelmann <james.grimmelmann@cornell.edu>, Yan Ji <jyamy42@gmail.com>, Tyler Kell <sk3259@cornell.edu>
+discussions-to: https://github.com/ethereum/eips/issues/5218
+created: 2022-7-11
+requires: 721
+---
+
+## Table of Contents ##
+
+* [Simple Summary](#simple-summary)
+* [Abstract](#abstract)
+* [Motivation](#motivation)
+* [Specification](#specification)
+* [Rationale](#rationale)
+* [Backwards Compatibility](#backwards-compatibility)
+* [Test Cases](#test-cases)
+* [Reference Implementation](#reference-implementation)
+* [Security Considerations](#security-considerations)
+* [Copyright](#copyright)
+* [Citation](#citation)
+
+## Simple Summary
+
+A standard interface for creating copyright licenses that transfer with an NFT.
+
+## Abstract
+
+The following standard defines an API for managing NFT licenses. This standard provides basic functionality to create, transfer, and revoke licenses, and to determine the current licensing state of an NFT. The standard does not define the legal details of the license. Instead, it provides a structured framework for recording licensing details.
+
+We consider use cases of NFT creators who wish to give the NFT holder a copyright license to use a work associated with the NFT. The license can optionally be revoked under conditions specified by the creator. The holder of an active license can issue sublicenses to others to carry out the rights granted under the license.
+
+
+## Motivation
+
+The [ERC-721](https://eips.ethereum.org/EIPS/eip-721) standard defines an API to track and transfer ownership of an NFT. When an NFT is to represent some off-chain asset, however, we would need some legally effective mechanism to *tether* the on-chain asset (NFT) to the off-chain property. One important case of off-chain property is a creative work such as an image or music file. Recently, most NFT projects involving creative works have used licenses to clarify what legal rights are granted to the NFT owner. But these licenses are almost always off-chain and the NFTs themselves do not indicate what licenses apply to them, leading to uncertainty about rights to use the work associated with the NFT. It is not a trivial task to avoid all the [copyright vulnerabilities in NFTs](https://medium.com/initc3org/copyright-vulnerabilities-in-nfts-317e02d8ae26), nor have existing EIPs addressed rights management of NFTs beyond the simple cases of [direct ownership](https://eips.ethereum.org/EIPS/eip-721) or [rental](https://eips.ethereum.org/EIPS/eip-4907).
```

Comment:
```suggestion
The [EIP-721](./eip-721.md) standard defines an API to track and transfer ownership of an NFT. When an NFT is to represent some off-chain asset, however, we would need some legally effective mechanism to *tether* the on-chain asset (NFT) to the off-chain property. One important case of off-chain property is creative work such as an image or music file. Recently, most NFT projects involving creative works have used licenses to clarify what legal rights are granted to the NFT owner. But these licenses are almost always off-chain and the NFTs themselves do not indicate what licenses apply to them, leading to uncertainty about rights to use the work associated with the NFT. It is not a trivial task to avoid all the copyright vulnerabilities in NFTs, nor have existing EIPs addressed rights management of NFTs beyond the simple cases of direct ownership (see [EIP-721](./eip-721.md)) or rental (see [EIP-4907](./eip-4907.md)).
```